examples/json_validation/main.go

```diff
@@ -45,7 +45,7 @@ type pluginConfiguration struct {
 // Override types.DefaultPluginContext.
 func (ctx *pluginContext) OnPluginStart(pluginConfigurationSize int) types.OnPluginStartStatus {
 	data, err := proxywasm.GetPluginConfiguration()
-	if err != nil {
+	if err != nil && err != types.ErrorStatusNotFound {
```

Comment:
cc @mathetake I think this is common enough that maybe shouldn't be an error? I don't know.

Reply:
this is lined with Proxy-Wasm's system call specification where not found is defined as error, not Go specific https://github.com/proxy-wasm/proxy-wasm-cpp-host/blob/master/src/exports.cc#L475-L480

Reply:
but yeah we could change and just return empty slice instead

Reply:
I mean, internally